src/parser/jobs/dnc/modules/Esprit.tsx

```diff
@@ -78,20 +61,20 @@ export default class EspritGauge extends Module {
 	private potentialOvercap = 0
 	private espritConsumed = 0
 	private avgGenerated = 0
-	private history: any[] = []
+	private history: Array<{t: number, y: number, isGenerator: boolean}> = [{t: 0, y: 0, isGenerator: false}]
```

Comment:
👍 

if you find yourself needing that type elsewhere, you can move it into an interface - but doesn't seem necessary in this case.